src/main/resources/application.yaml

```diff
@@ -515,6 +515,7 @@ hearingCentreTelephoneNumber:
 endAppealHomeOfficeEmailAddress: ${IA_HOME_OFFICE_END_APPEAL_EMAIL:ho-end-appeal@example.com}
 allowedAppealHomeOfficeEmailAddress: ${IA_HOME_OFFICE_ALLOWED_APPEAL_EMAIL:ho-allowed-appeal@example.com}
 dismissedAppealHomeOfficeEmailAddress: ${IA_HOME_OFFICE_DISMISSED_APPEAL_EMAIL:ho-dismissed-appeal@example.com}
+apcPrivateHomeOfficeEmailAddress: ${IA_HOME_OFFICE_APC_EMAIL:apc-submit-appeal@example.com}
```

Comment:
please do not add default value to the application.yaml, lets keep it as:
```
apcPrivateHomeOfficeEmailAddress: ${IA_HOME_OFFICE_APC_EMAIL}
```
it is latest recommendation from Platform Engineering Team to not add default values for secrets stored in vault.